WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -745,6 +747,20 @@ extension OrderDetailsViewModel {
         stores.dispatch(deleteTrackingAction)
     }
 
+    /// Put an order in the trash, without deleting it permanently.
+    ///
+    func trashOrder(_ onCompletion: @escaping (Result<Order, Error>) -> Void) {
```

Comment:
Nice to have: We could make this an `async` method and use `await` to call this. It will make the code and error handling simpler to read.